### DISCUSSION TITLE
Default computer core?

### DISCUSSION THREAD

**Discussion by hhy2022** [idaholab/moose#23675] [Q&A General] [answered]
Hi, I am trying to figure out the computational expense of the calculation. When I am runing a case, I use the command 'myapp-opt -i input.i' without any other specification. Does it mean I am using the default core? How many computer core I am using? Can I use more cores to shorter the calculation time?

Thank you very much for any help.

**Comment by Rahim-Habibi** [comment, marked as answer]
Hi, 
You can use following command to set core numbers, the nomber after "n" shows core numbers, here for example it is 6 : 
mpirun -n 6 ./AppName-opt -i input.i

Rahim

**Reply by hhy2022** [reply]
Thank you so much for the information.
Besides, do you have any idea on how to check the core number I am using now? I didn't specify any core numbers, just want to know the defult value (if there was one).

**Reply by GiudGiud** [reply]
In the calculation header you can look at the number of processes used.
By default you are using 1

**Reply by hhy2022** [reply]
Got it. 
Thank you very much!
